.devcontainer/src/test/regress/Pipfile

```diff
@@ -16,6 +16,7 @@ pytest-timeout = "*"
 pytest-xdist = "*"
 pytest-repeat = "*"
 pyyaml = "*"
+werkzeug = "==2.3.7"
```

Comment:
Guess once this was added the file hasn't been copied to the devcontainer. Guess thats an extra CI job to setup :P. Not super important, but now it will get incorporated with this change.